PedDetection.md

```diff
@@ -59,30 +59,35 @@ The pedestrian detection system shall be capable of detecting the presence of pe
+   1. Please provide information about methods of remotely updating Firmware in a secure manner. Does your solution support failsafe firmware updates? Can OS updates be applied independently of application updates?
+   1. Please provide information about your ability to remotely add additional analytics. Can the proposed solution support detection of other classes of objects? Can these applications be securely deployed remotely?
+
 1. Video Outputs:  The pedestrian detection system manufacturer shall indicate what items it supports and does not support from the list below.  The manufacturer is invited to also list items it supports that are not listed below that could be beneficial for CCD.
    1. Digital:  H.264/H.265
    1. Streaming:  Maximum Bit Rate (MBR), Variable Bit Rate (VBR), and/or Constant Bit Rate (CBR) from 50 Kbps up to 4 Mbps
    1. Video:  Real-time Transport Protocol (RTP)/Real Time Streaming Protocol (RTSP), unicast/multicast
    1. Display:  For video, the pedestrian detection system manufacturer shall describe the number of colors its system can support for output displays.  For thermal, the manufacturer shall describe how warmer objects will be displayed with cooler objects and the number of colors available in its color palette.
    1. Interface:  Ethernet, RJ-45 (8P8C connector, T568B pinout), minimum 10/100 Mbps
-   1. Supported Services and Protocols:  Address Resolution Protocol (ARP), Domain Name System (DNS), Dynamic Host Configuration Protocol (DHCP), File Transfer Protocol (FTP), HyperText Transfer Protocol (HTTP), HyperText Transfer Protocol Secure (HTTPS), Internet Control Message Protocol (ICMP), Internet Group Management Protocol (IGMP), Internet Protocol version 4 (IPv4), Network Time Protocol (NTP), RTP Control Protocol (RTCP), Secure Copy Protocol (SCP), Secure Shell (SSH), Transmission Control Protocol (TCP), Universal Plug and Play (UPnP), and User Datagram Protocol (UDP)
+   1. Supported Services and Protocols:  Address Resolution Protocol (ARP), Domain Name System (DNS), Dynamic Host Configuration Protocol (DHCP), File Transfer Protocol (FTP), HyperText Transfer Protocol (HTTP), HyperText Transfer Protocol Secure (HTTPS), Internet Control Message Protocol (ICMP), Internet Group Management Protocol (IGMP), Internet Protocol version 4 (IPv4), Network Time Protocol (NTP), RTP Control Protocol (RTCP), Secure Copy Protocol (SCP), Secure Shell (SSH), Transmission Control Protocol (TCP), Universal Plug and Play (UPnP), and User Datagram Protocol (UDP), National Transportation Communications for Intelligent Transportation (NTCIP), Simple Network Management Protocol (SNMP), and Message Queuing Telemetry Transport (MQTT). 
```

Comment:
@MatthewJMcAllister consider removing the entire section for supported services and protocols before posting and converting to a question.